src/Calculator/Views/GraphingCalculator/GraphingNumPad.xaml.cpp

```diff
@@ -18,21 +20,105 @@ using namespace Windows::UI::Xaml::Input;
+    { NumbersAndOperatorsEnum::CloseParenthesis, { L")", 1 } },
+    { NumbersAndOperatorsEnum::Equals, { L"=", 1 } },
+    { NumbersAndOperatorsEnum::Divide, { L"/", 1 } },
+    { NumbersAndOperatorsEnum::Multiply, { L"*", 1 } },
+    { NumbersAndOperatorsEnum::Subtract, { L"-", 1 } },
+    { NumbersAndOperatorsEnum::Add, { L"+", 1 } },
+    { NumbersAndOperatorsEnum::Invert, { L"1/", 2 } },
+    { NumbersAndOperatorsEnum::Negate, { L"-", 1 } },
+    { NumbersAndOperatorsEnum::GreaterThan, { L">", 1 } },
+    { NumbersAndOperatorsEnum::GreaterThanOrEqualTo, { L"\u2265", 1 } },
+    { NumbersAndOperatorsEnum::LessThan, { L"<", 1 } },
+    { NumbersAndOperatorsEnum::LessThanOrEqualTo, { L"\u2264", 1 } },
+    { NumbersAndOperatorsEnum::Zero, { L"0", 1 } },
+    { NumbersAndOperatorsEnum::One, { L"1", 1 } },
+    { NumbersAndOperatorsEnum::Two, { L"2", 1 } },
+    { NumbersAndOperatorsEnum::Three, { L"3", 1 } },
+    { NumbersAndOperatorsEnum::Four, { L"4", 1 } },
+    { NumbersAndOperatorsEnum::Five, { L"5", 1 } },
+    { NumbersAndOperatorsEnum::Six, { L"6", 1 } },
+    { NumbersAndOperatorsEnum::Seven, { L"7", 1 } },
+    { NumbersAndOperatorsEnum::Eight, { L"8", 1 } },
+    { NumbersAndOperatorsEnum::Nine, { L"9", 1 } },
+    { NumbersAndOperatorsEnum::Decimal, { L".", 1 } },
+};
+
 GraphingNumPad::GraphingNumPad()
 {
     InitializeComponent();
+    const auto& localizationSettings = CalculatorApp::Common::LocalizationSettings::GetInstance();
+    DecimalSeparatorButton->Content = localizationSettings.GetDecimalSeparator();
+    Num0Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('0');
+    Num1Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('1');
+    Num2Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('2');
+    Num3Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('3');
+    Num4Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('4');
+    Num5Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('5');
+    Num6Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('6');
+    Num7Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('7');
+    Num8Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('8');
+    Num9Button->Content = localizationSettings.GetDigitSymbolFromEnUsDigit('9');
 }
 
 void GraphingNumPad::ShiftButton_Check(_In_ Platform::Object ^ /*sender*/, _In_ Windows::UI::Xaml::RoutedEventArgs ^ /*e*/)
 {
     SetOperatorRowVisibility();
 }
 
-void GraphingNumPad::ShiftButton_Uncheck(_In_ Platform::Object ^ /*sender*/, _In_ Windows::UI::Xaml::RoutedEventArgs ^ /*e*/)
+void GraphingNumPad::ShiftButton_Uncheck(_In_ Platform::Object ^ sender, _In_ Windows::UI::Xaml::RoutedEventArgs ^ /*e*/)
 {
     ShiftButton->IsChecked = false;
     SetOperatorRowVisibility();
-    ShiftButton->Focus(::FocusState::Programmatic);
+
+    GraphingNumPad::Button_Clicked(sender, nullptr);
```

Comment:
I think it's most likely a focus issue... I'll debug.

Reply:
This is because of how Rudy's focus change and my change interact.  I think I have a few ideas on how to fix it.

Reply:
Nevermind.  I missed AllowFocusOnInteraction="False" on the flyout.  will fix with the next iteration.

Reply:
Added all of the xaml to prevent focus from being caught, and then added a border element to the graphing num pad to swallow unhandled clicks (which cover both the open space near the flyouts as well as the pixels in between buttons).